pages/dkp/kommander/1.3/clusters/attach-cluster/index.md

```diff
@@ -9,15 +9,17 @@ excerpt: A guide for attaching an existing Kubernetes cluster using kubeconfig
 
 ## Attach Kubernetes Cluster
 
-You can attach an existing cluster directly to Kommander. If the cluster you want to attach was created using Amazon EKS, Azure AKS, or Google GKE, create a service account as described below. If you are attaching an Amazon EKS cluster to Kommander, [detailed instructions are available](/dkp/kommander/latest/clusters/attach-cluster/attach-eks-cluster/).
+You can attach an existing cluster directly to Kommander. At attachment time certain namespaces are created on the cluster, and workspace platform services are deployed automatically into the newly-created namespaces. Review the [workspace platform service resource requirements][platform_service_req] to ensure the attached cluster has sufficient resources. For more information on platform services and customizing them, see [workspace platform services][workspace_platform_services].
```

Comment:
Would this and https://github.com/mesosphere/dcos-docs-site/pull/3400/commits/0eb9bafa1840de5cf5919e54569654634b702d58 suffice for https://jira.d2iq.com/browse/COPS-6437 ? I didn't think we needed to include the specifics here as it may be too much detail and also subject to change (e.g. the namespaces that are created)

Reply:
I think that would work!